app/uk/gov/hmrc/fileupload/read/routing/model.scala

```diff
@@ -78,10 +78,14 @@ object FileTransferNotification {
     implicit val auditFormat =
       (__ \ "correlationID").format[String].inmap(Audit.apply, unlift(Audit.unapply))
 
+    implicit val downloadUrlFormat =
+      (__ \ "value").format[String].inmap(DownloadUrl.apply, unlift(DownloadUrl.unapply))
```

Comment:
this is a change in Json format. I think you just need the following below
```scala
(__ \ "location" ).formatNullable[String].inmap[DownloadUrl](...)
```

Reply:
Oh whoops, sorry I missed that.

@jonnydh: Or if you did want to define a `Format[DownloadUrl]` to reuse across the `Format[ZipData]` and `Format[FileTransferNotification]` instances, you could do something like:

```scala
  object DownloadUrl {
    implicit val format: Format[DownloadUrl] =
      implicitly[Format[String]].inmap(DownloadUrl(_), _.toString)
  }
```